### DISCUSSION TITLE
Any Plans to make it work without AWS

### DISCUSSION THREAD

**Discussion by llPekoll** [usesend/useSend#79] [Ideas]
Hi guys,

I know that Unsend is a wrapper around SES but, any Plans to make it work without AWS.
It would be nice to make it work bith just coolify, with an AWS
it could run 100% open source then

Best

**Comment by KMKoushik** [comment]
not at the moment and if someone has any idea to make this work i'm all ears. basically it means we need to do bunch of domain/hooks stuff that is supported by aws

**Comment by mo9a7i** [comment]
The pain here will be running local mail servers, which will be bound to the IPs you get with the server, and then, one will have to maintain good IP **reputation** which is not easy and any spam report could damage the reputation easily. Having it with cloud providers such AWS, GCP, Azure lifts this headache to the providers, who unfortunately get good reputation without doing anything anyways.

I would vote for other providers integrations in the future, rather than running it locally.

**Comment by Stupremee** [comment]
Hi,

how about starting to support non-AWS cloud services. Specifically Azure Communication Service.
I would love to use useSend, however, we use Azure in our company so currently we send all mails via ACS.

If I find some time, I might even try to open a pull request to support Azure Communication Service, in combination with some sort of generic sender interface if you are open to this feature.

**Comment by KMKoushik** [comment]
it needs quiet a work to support it, if someone put the effort on pr, i'll try to review and merge it

**Comment by zdeneksvarc** [comment]
If someone does not want to use AWS or any other cloud provider as an essential component, which I completely understand, they can use [Postal](https://github.com/postalserver/postal).
